docs/source/nlp/usage_guides/nlp_embeddings.rst

```diff
@@ -0,0 +1,129 @@
+.. _nlp__embeddings_guide:
+
+=================
+NLP Embeddings
+=================
+
+Embeddings are a way to represent text as a vector of numbers. The vector is a representation of the text in the latent
+space (or semantic space), in which text with similar meaning is represented by similar vectors.
```

Comment:
Isn't semantic space only relevant when the model is trained on a semantic task? What if the model is trained to recognize language for e.g.?

Reply:
Fixed